rpc/client.go

```diff
@@ -43,7 +43,7 @@ var (
 const (
 	// Timeouts
 	defaultDialTimeout = 10 * time.Second // used if context has no deadline
-	subscribeTimeout   = 5 * time.Second  // overall timeout eth_subscribe, rpc_modules calls
+	subscribeTimeout   = 10 * time.Second // overall timeout eth_subscribe, rpc_modules calls
```

Comment:
This change looks like it not only concerns tests, but actually modifies production code?

Reply:
Yeah, BTW, it's strange that the dial timeout is set to 10 seconds, but the subscribe timeout is only set to 5 seconds. So I set it to the same as dial's.